Fakebook.Profile/Fakebook.Profile.DataAccess/ProfileRepository.cs

```diff
@@ -66,12 +126,20 @@ private static EntityProfile ToEntityProfile(DomainProfile profile)
                 LastName = profile.LastName,
                 PhoneNumber = profile.PhoneNumber,
                 BirthDate = profile.BirthDate,
-                Status = profile.Status
+                Status = profile.Status,
+                Following = followingEmails,
+                Followers = followerEmails
             };
 
             return convertedProfile;
         }
 
+        private async Task<int> GetProfileIdAsync(string email)
+        {
+            var entity = await _context.EntityProfiles.FirstAsync(x => x.Email == email);
+            return entity.Id;
+        }
```

Comment:
if you write this as `await _context.EntityProfiles.Where(x => x.Email == email).Select(x => x.Id).FirstAsync()` then we won't have to query the whole row, just the ID.

Reply:
Done